kouta-backend/src/main/scala/fi/oph/kouta/servlet/HakuServlet.scala

```diff
@@ -137,9 +144,10 @@ class HakuServlet(hakuService: HakuService) extends KoutaServlet {
 
     implicit val authenticated: Authenticated = authenticate()
 
-    params.get("organisaatioOid").map(OrganisaatioOid) match {
-      case None => NotFound()
-      case Some(oid) => Ok(hakuService.list(oid))
+    (params.get("organisaatioOid").map(OrganisaatioOid), params.get("myosArkistoidut")) match {
+      case (None, _) => NotFound()
+      case (Some(oid), None) => Ok(hakuService.list(oid, myosArkistoidut = true))
+      case (Some(oid), Some(myosArkistoidut)) => Ok(hakuService.list(oid, myosArkistoidut.toBoolean))
```

Comment:
Tästä saisi yhden rivin pois ja tulisi luettavampi näin:

```
(params.get("organisaatioOid").map(OrganisaatioOid), params.getOrElse("myosArkistoidut", "true").toBoolean) match {
      case (None, _) => NotFound()
      case (Some(oid), myosArkistoidut) => Ok(hakuService.list(oid, myosArkistoidut))
    }
```

Reply:
Totta, yritin itsekin saada yhdelle riville mutta taisin sitten kompastua tuohon että en tajunnut laittaa getOrElselle defaultia stringinä.